siac/walletcmd.go

```diff
@@ -97,6 +90,20 @@ The seed string, which is also the encryption password, will be returned.`,
 		Run:   wrap(walletload033xcmd),
 	}
 
+	walletLoadseedCmd = &cobra.Command{
+		Use:   `loadseed`,
+		Short: "Add a seed to the wallet",
+		Long:  "Uses the given password to create a new wallet with that as the primary seed",
+		Run:   wrap(walletloadseedcmd),
+	}
+
+	walletLoadsiagCmd = &cobra.Command{
```

Comment:
Oh. I had a different interpretation for siac: Capitalize with each new command.

IE:

siac dog cat manbearpig => dogCatManbearpigCmd

I do agree that `load` should become its own command, but I didn't know how to do it so I left it for you :P